tests/integration/test_storage_rabbitmq/test.py

```diff
@@ -35,6 +35,28 @@ def rabbitmq_check_result(result, check=False, ref_file='test_rabbitmq_json.refe
         else:
             return TSV(result) == TSV(reference)
 
+def check_rabbitmq_is_available(rabbitmq_id):
+    p = subprocess.Popen(('docker',
+                        'exec',
+                        '-i',
+                        rabbitmq_id,
+                        'rabbitmqctl',
+                        'await_startup'),
+                        stdout=subprocess.PIPE)
+    p.communicate()
+    return p.returncode == 0
+
+def wait_rabbitmq_to_start(rabbitmq_docker_id, timeout=180):
```

Comment:
Why helper.py methods are not sufficient? 
I don't like to copy them in tests. We should try to use common interfaces, so it is easier to maintain.